js/core/utils/dom.js

```diff
@@ -198,6 +198,50 @@ setPublicElementWrapper(function(element) {
     return element && element.get(0);
 });
 
+var calculateLimitHeight = function(value, container, offset, defaultValue) {
+    if(!value) {
+        return defaultValue;
+    }
+
+    if(value === "auto" || value === "none" || value === "inherit" || value === "initial") {
+        return offset ? defaultValue : value;
+    }
+
+    if(typeof value === "string") {
+        if(value.indexOf("px") > 0) {
```

Comment:
Is it works well with the other [length values](https://www.w3.org/TR/CSS2/syndata.html#value-def-length)?
E.g. `em`, `cm`, `pt` etc.

Reply:
Yes, it generates `calc` expression for CSS. You can see some results in utils tests